praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/MeldezeitraumController.java

```diff
@@ -58,6 +60,7 @@ public List<MeldezeitraumDto> getMeldezeitraeume(@RequestParam(name = "period",
     @DeleteMapping
     @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
     public void deleteMeldezeitraum(@RequestParam(name = "id") UUID id) {
-        meldezeitraumService.deleteMeldezeitraumAndAttachedPraktikumsstellen(id);
+        praktikumsstellenService.deleteAllPraktikumsstellenByMeldezeitraumId(id);
+        meldezeitraumService.deleteMeldezeitraumById(id);
```

Comment:
when an exception occurs during deleteMeldezeitraum the stellen will remain deleted. Is that intended?

Reply:
this is not intended, and fixed with 3316054